lib/ecto/postgres/pg_rational_migrations.ex

```diff
@@ -0,0 +1,209 @@
+use Vtc.Ecto.Postgres.Utils
+
+defpgmodule Vtc.Ecto.Postgres.PgRational.Migrations do
+  @moduledoc """
+  Migrations for adding rational types, functions and constraints to a
+  Postgres database.
+  """
+  alias Vtc.Ecto.Postgres
+
+  @doc section: :migrations_full
+  @doc """
+  Adds raw SQL queries to a migration for for creating the database types, associated
+  functions, casts, operators, and operator families.
+
+  This migration included all migraitons under the `PgTypes` and `PgFunctions` headings.
```

Comment:
```suggestion
  This migration included all migrations under the `PgTypes` and `PgFunctions` headings.
```

...not sure what the PgTypes and PgFunctions headings are? This line may could just be removed.

Reply:
It's referring to sections in the rendered documentation:

<img width="296" alt="Screen Shot 2023-06-19 at 2 21 23 PM" src="https://github.com/opencinemac/vtc-ex/assets/6580550/57996b6f-647d-4b78-8b4b-d26e4a4e983d">

I made these explicit links here: 6d35c768a8fd316b0821434b68287f9827325346

Reply:
ah thanks! learning some new things as to how elixir libraries create documentation via this PR.